LevelComponents/Room.cpp

```diff
@@ -50,8 +50,6 @@ namespace LevelComponents
             int layerPtr = ROMUtils::PointerFromData(roomDataPtr + i * 4 + 8);
             layers[i] = new Layer(layerPtr, mappingType);
         }
-        Layer0width = layers[0]->GetLayerWidth();
```

Comment:
these are member variables i added to fix Layer0 render problem for edge cases, cannot delete

Reply:
i fixed it by myself